includes/Modules/Search_Console.php

```diff
@@ -418,18 +403,20 @@ function( $dataset ) {
 			);
 
 			$has_data = false;
-			foreach ( $responses as $key => $response ) {
-				if ( is_wp_error( $response ) || ! is_array( $response ) || empty( $response ) || ! isset( $response[0] ) ) {
+
+			// Go thru results, any impressions means the URL has data.
+			foreach ( $responses['sc-site-analytics'] as $key => $response ) {
+				if ( is_wp_error( $response ) || empty( $response ) || ! isset( $response ) ) {
```

Comment:
Good catch! This is necessary because of the changed response format right?

Reply:
Apparently so! When I was testing the `impressions` I was looking for were directly on `$response` not `$response[0]` as in the previous code. My assumption was this had changed in the response and hadn't been adjusted here yet.